sdr_paper/sdr_calculations.cpp

```diff
@@ -222,7 +236,7 @@ void classificationFalseMatchProbability(
     {
       if (matchesWithThetas[theta] > 0)
```

Comment:
We don't need the if condition do we? We can just put `probWithThetas[theta] += matchesWithThetas[theta];`